### DISCUSSION TITLE
Difference in the coordinates given by MDAnalysis and LAMMPS dump file

### DISCUSSION THREAD

**Discussion by sanjeets47** [MDAnalysis/mdanalysis#4452] [Q&A]
Hello Group Members,

I am trying to perform some analysis in MDAnalysis. For this, I need to extract the coordinate information of atoms with atom type 4 from my LAMMPS dump file. But when I try to match the coordinates from MDAnalyis to that of the LAMMPS dump file, I see that the coordinates are not the same.

My LAMMPS dump file is in custom style with the following format:

```
dump        trj all custom 500 500-Trajectory.lammpsdump id mol type q xu yu zu ix iy iz vx vy vz
```
And I am trying to load the trajectory in MDAnalysis with the following command:
```python
u = mda.Universe("PE.data",500-Trajectory.lammpsdump", atom_style='id resid type charge x y z')
```
For extracting the atoms with atom type 4 with specific resid, I am using the following command:
```python
ag = u.select_atoms('resid %d:%d and type 4' %(i,i))
```
Can you please point out the reason for this discrepancy?

I appreciate your help.

Thank you,
Sanjeet

**Comment by orbeckst** [comment]
How do the coordinates differ? Can you show actual examples with the output from MDAnalysis and your original output.

Have you tried looking at which coordinates agree to see if the resids are perhaps off by 1?

If you just look at the very first coordinate:
```
u.atoms[0].positions
```
and compare to what the dump file shows, what do you get?
